creator-node/src/AsyncProcessingQueue.ts

```diff
@@ -243,11 +271,20 @@ class AsyncProcessingQueue {
    * @param {Object} param.req the request associated with the func
    * @returns the expected response from the func
    */
-  async monitorProgress(task, func, { logContext, req }) {
+  async monitorProgress(
+    task: string,
+    func: (...args: any) => any,
+    { logContext, req }: { logContext: any; req: Request }
```

Comment:
i think when i get the `CustomRequest` type merged, we can update the type `Request` to `CustomRequest`

Reply:
Yes, for sure. The `CustomRequest` type will be amazing

Reply:
Yup, totally down